proxy-app/errors/error-handler.js

```diff
@@ -0,0 +1,20 @@
+const {customError} = require("./errors");
+
+const processAndSendError = (err, resp) => {
```

Comment:
Sounds like your method breaks the first point in SOLID methodology 😁
Probably, you can change the name just to `processError()`

Reply:
Was renamed, tnx!